orchestration/templates/ingest-xml-archive.yaml

```diff
@@ -367,3 +368,28 @@ spec:
             json='{"release_date":"{{ $releaseDate }}","archive_path":"{{ $archiveId }}"}'
             echo "$json" > archive.json
             gsutil cp archive.json {{ $uploadPath }}
+
+    ##
+    ## Send a Slack notification describing the final status of the workflow.
+    ##
+    - name: send-slack-notification
```

Comment:
Yes, this is copy pasted. No, it is not DRY. Keeping Argo yaml DRY is a time sink, so if anyone feels strongly about this, I can make it DRY, but be warned that it'll take longer than it should.

Reply:
We have a [template](https://github.com/broadinstitute/monster-helm/blob/master/charts/argo-templates/templates/notification.yaml) for this in monster-helm already.  For some reason I thought we had switched over to using it, but evidently not. Might be worth spending 10 minutes and seeing if it'll work ok here? Not a blocker for this PR though.